meetings/2024-01-17.md

```diff
@@ -0,0 +1,126 @@
+
+## Links
+
+* **Recording**: <https://youtube.com/live/00oahsrkFTU>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1492>
+
+## Present
+
+* Antoine du Hamel @aduh95 (voting member)
+* Yagiz Nizipli @anonrig (voting member)
+* Benjamin Gruenbaum @benjamingr (voting member)
+* Ruben Bridgewater @BridgeAR (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* Gireesh Punathil @gireeshpunathil (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Matteo Collina @mcollina (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Richard Lau @richardlau (voting member)
+* Ruy Adorno @ruyadorno (voting member)
+* Michaël Zasso @targos (voting member)
+* Claudio Wunder (Guest)
+* Steven @styfle (Guest)
+* Darcy Clarke @darcyclarke (Guest)
+* Mael Nison (Guest)
+
+## Agenda
+
+### Announcements
+
+* No announcements this week
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* From Claudio
+  * We're closing in on finishing the Travel Fund Rework
+  * Same for the CoC rework
+  * We're applying the Foundation (OpenJS) to Google Season of Docs -- to have Node.js as a participating project.
+
+### nodejs/node
+
+* enable corepack by default [#50963](https://github.com/nodejs/node/issues/50963)
+  * Antoine - issue opened last month, asking if we can enable by default. Quite a few
+    discussions that have come from that, one being the integration of npm.
+  * Darcy - <https://github.com/nodejs/node/pull/35398#issuecomment-700314358> included a
+    number of questions that would be answered before we would unflag. Would consider that a
+    blocker to unflagging.
+  * Mael - as package manager for yarn, has definitely solved the problem they had before in
+    terms of using the package manager that they.
+  * Michael, one option not discussed too much, mark stable but still require corepack enable,
+    * Mael might be ok, but what is the point of forcing the extra command
+    * Michael to try to make sure they know they are enable auto install of software for them
+  * Darcy - link that shows registry traffic
+    * Sharing some data sourced from the public npm team statements: <https://github-production-user-asset-6210df.s3.amazonaws.com/459713/297423226-1d18f2dc-1deb-457b-8a14-5f512921f8cc.png>
+    * there was almost a 50/50 split at one point, which shows that blockers of consumption might
+      not have been there original
+  * Steven - data shows us that people are using different package managers, the problem was
+    not that. The problem that corepack is solving is helping a new user get up and running
+    Faster.
+* Steven to answer the question about corepack by default. Seems like it should be easy.
+  Because it’s not enable by default Vercel does not have it enabled by default
+  * there are github actions where it will not be enabled because it is not enabled by default.
```

Comment:
```suggestion
  * the official [@action/setup-node](https://github.com/actions/setup-node) where it will not be enabled because it is not enabled by default. There have been [multiple PRs](https://github.com/actions/setup-node/pulls?q=is%3Apr+corepack) to add support for corepack.
```